README.md

```diff
@@ -96,25 +96,39 @@ const SimpleFormApp = () => (
 ## Release Managament
 
 - Starting with `1.0.0` all future releases will follow semver semantics:
+
   - `patch` (eg: 1.0.x) for **API compatible bug fixes**
   - `minor` (eg.: 1.x.0) for **API compatible new features**
   - `major` (eg: 2.0.0) for **API breaking changes**
-  
+
 - For early integration of upcoming release changes we use the following conventions:
-  
-  - `[current version]-next.x` to tag changes for upcoming releases (as we cannot know the necessary 
-    semver for the final release including all the changes). `x` in this case is simply a number that 
+
+  - `[current version]-next.x` to tag changes for upcoming releases (as we cannot know the necessary
+    semver for the final release including all the changes). `x` in this case is simply a number that
     is increased and can be thought of like "slots" for temporary releases
-    
-  - All temporary releases should be published with the `next` npm dist tag via: `npm publish --tag next` 
+
+  - All temporary releases should be published with the `next` npm dist tag via: `npm publish --tag next`
     so that they are not automatically tagged with the default `latest` npm tag.
-    
-- The `master` branch only includes commits of final releases
-  
-- `release/x.x.x` branches are created as soon as we cut a release and know the correct semver - they 
-  are always targeting the `master` branch + should be well documented. They can include many release
-  candidates which should be tagged like `[next releaes]-rc.X` where you increment X per release candidate
-  until we are confident that the release is ready to be published under its normal version.
+
+### The complete development flow considering the Daedalus
+
+1. You develop a feature on your branch without bumping the version
+2. Once your code is ready to be tested in Daedalus, you [publish a temporary release](#how-to-publish-a-temporary-release):
+
+- checkout to your branch:
+- change the version in package json to `[current version]-next.x` (only add the -next.X suffix keeping the current version)
+- publish the version under the next tag
+- **don't push the version change to upstream**
```

Comment:
@szymonmaslowski could you please elaborate on this one? What could happen in case we push ?

Reply:
Nothing 😂 Maybe it should be rephrased to `you don't need to push the version change to upstream as you will be changing it anyway`, what do you think?